icons/qwik.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Qwik</title><path d="m20.55562 24-4.30941-4.28502-.06585.0094v-.04699L7.0158 10.6281 9.274 8.448 7.94732.83648 1.65257 8.63596C.57992 9.7166.38233 11.47384 1.15388 12.77061l3.93304 6.52152c.60219 1.00547 1.69364 1.60687 2.86981 1.59746l1.94772-.01881L20.55562 24Zm2.35195-15.51446-.86567-1.59747-.45166-.81752-.17874-.31951-.01881.01881-2.37115-4.10648C18.42877.6297 17.3185-.0093 16.11412.0001l-2.07943.05638-6.2007.0188c-1.17611.0094-2.24875.6296-2.84155 1.63507L1.21936 9.19032 7.96576.81763l8.84465 9.71644-1.57138 1.58811.94094 7.60215.0094-.01881v.01881h-.0188l.0188.01877.73393.71419 3.56608 3.48627c.15057.14096.39521-.02818.29168-.20672l-2.20172-4.33202 3.83895-7.09469.1223-.14096c.04703-.0564.09407-.11274.13175-.16914.75274-1.0243.85622-2.39622.23523-3.5145Z"/></svg>
```

Comment:
When comparing with the logo from their website header, I got the following differences:
![image](https://github.com/simple-icons/simple-icons/assets/12817388/70cf6209-0aa5-42f3-b66f-71e70479dcf7)

I found this logo in their Github repository that might be a better source since there are no imperfections:
https://github.com/BuilderIO/qwik/blob/c88e53d49dc65020899d770338f4e51f3134611e/packages/docs/public/logos/qwik-logo.svg